appeals/api/src/server/repositories/document.repository.js

```diff
@@ -20,14 +20,34 @@ export const getDocumentById = (guid) => {
 		include: {
 			latestDocumentVersion: {
 				include: {
-					avScan: true,
 					redactionStatus: true
 				}
 			}
 		}
 	});
 };
 
+/**
+ * @param {string} guid
+ * @param {Number} version
+ * @returns {PrismaPromise<Document|null>}
+ */
+export const getDocumentByIdAndVersion = (guid, version) => {
+	return databaseConnector.document.findUnique({
+		where: { guid },
+		include: {
+			versions: {
+				include: {
+					redactionStatus: true
+				},
```

Comment:
Does this mean it won't update virus scan status unless it's redacted?

Reply:
no, actually that's to bring the redactionStatus data - but is not needed in this call. I'll remove

Reply:
done

Reply:
Ah I misunderstood, not a where but an include clause!